presto-hive/src/test/java/com/facebook/presto/hive/statistics/TestMetastoreHiveStatisticsProvider.java

```diff
@@ -116,6 +118,14 @@ public void testGetPartitionsSample()
         assertEquals(getPartitionsSample(ImmutableList.of(p1, p2, p3, p4, p5), 3), ImmutableList.of(p1, p5, p4));
     }
 
+    @Test
+    public void testNullablePartitionValue()
+    {
+        HivePartition part = partition("p1=name/p2=\\N");
+        NullableValue verify = new NullableValue(BIGINT, null);
+        assertTrue(part.getKeys().values().contains(verify));
```

Comment:
`containsValue`

Reply:
Collection of NullableValue for partition key does not implement containsValue method, but contains method works fine

Reply:
What I meant is to change `values().contains()` to `containsValue`

Reply:
fixed